Source/GoBot/ViewDatabase.swift

```diff
@@ -2340,157 +2340,155 @@ class ViewDatabase {
         
         var reports = [Report]()
         var skipped: UInt = 0
-        try db.transaction { // also batches writes! helps a lot with perf
```

Comment:
This is really the only important change to this file. This line (and it's closing brace) were removed. Everything else was an indentation change, but Github doesn't show it that way.